chirho/observational/handlers/soft_conditioning.py

```diff
@@ -254,7 +205,7 @@ class AutoSoftConditioning(pyro.infer.reparam.strategies.Strategy):
         the site's ``event_dim`` and ``support``.
     """
 
-    def __init__(self, *, scale: float = 1.0, alpha: float = 1.0):
+    def __init__(self, *, scale: float = 1.0, alpha: float = 1e-8):
```

Comment:
I think you can delete the `alpha` keyword argument from `AutoSoftConditioning`, since `scale` is the only parameter used by `soft_eq` and we're delegating the semantics of soft conditioning entirely to `soft_eq`.

Reply:
Sure, will do - but keep in mind this means that even with a boolean constraints, `AutoSoftConditioning` will use the default scale of 1, which doesn't result in intuitively useful shape of values for this constraint.